lib/internal/modules/esm/loader.js

```diff
@@ -18,10 +19,10 @@ const {
 const { getOptionValue } = require('internal/options');
 const { pathToFileURL } = require('internal/url');
 const { emitExperimentalWarning } = require('internal/util');
-
 const {
   getDefaultConditions,
 } = require('internal/modules/esm/utils');
+let defaultResolve, defaultLoad, importMetaInitializer;
```

Comment:
ESLint does a better job explaining the rationale: https://eslint.org/docs/latest/rules/one-var

Maybe I'm just accustomed to seeing them declared separately.

Reply:
> It's better in case you're stepping through with the debugger.

Just to clarify, that's valid only for variable declarations that have a initializer, right? FWIW I agree it's better to keep them as separate instructions when the variables are initialized.